src/site/content/en/blog/generic-sensor/index.md

```diff
@@ -0,0 +1,478 @@
+---
+title: Sensors For The Web
+subhead: Use the Generic Sensor API to enable advanced features of your app
+authors:
+  - alexshalamov
+  - pozdnyakov
+  - thomassteiner
+description: |
+  Sensors are used in many platform-specific applications to enable advanced features.
+  Wouldn't it be nice to bridge the gap between native and the web?
+  You can do it with Generic Sensor API, which is enabled by default as of Chrome 67.
+date: 2017-09-18
+updated: 2021-02-05
+tags:
+  - blog
+  - sensors
+  - capabilities
+hero: # ⚠️ [TODO] Fix hero, old hero was "/web/updates/images/generic/screen-rotation.png"!
+alt: # ⚠️ [TODO] Add alt text!
+---
+
+Today, sensor data is used in many platform-specific applications to enable use cases such
+as immersive gaming, fitness tracking, and augmented or virtual reality. Wouldn't
+it be cool to bridge the gap between platform-specific and web applications? The [Generic Sensor
+API](https://www.w3.org/TR/generic-sensor/), for the web!
+
+## What is the Generic Sensor API? {: #what-is-generic-sensor-api }
+
+The [Generic Sensor API](https://www.w3.org/TR/generic-sensor/) is a set of
+interfaces which expose sensor devices to the web platform. The API consists
+of the base [Sensor](https://w3c.github.io/sensors/#the-sensor-interface)
```

Comment:
```suggestion
of the base [`Sensor`](https://w3c.github.io/sensors/#the-sensor-interface)
```
Please format these all as code since they are (presumably) actual interfaces that you can access in JS. It seems like `Gyroscope`, `AbsoluteOrientation`, etc. should also be formatted as code for the same reason